### ISSUE TITLE
Metro windows reload everytime the PC is unlocked

### ISSUE THREAD

**Issue by mzomparelli** [mzomparelli/zVirtualDesktop#52]
Even after closing them they come back every time I unlock. These metro apps are retarded. Thanks Microsoft for forcing me to have certain windows open and running even if I never plan on using them.

**Comment by mzomparelli** [comment]
Here is how you can uninstall some of those metro apps that Windows won't let you uninstall normally.

Open Power Shell as an admin and run any of the below commands. You should be able to tell which app each command is referring to.

```
Get-AppxPackage *3dbuilder* | Remove-AppxPackage
Get-AppxPackage *windowsalarms* | Remove-AppxPackage
Get-AppxPackage *windowscalculator* | Remove-AppxPackage
Get-AppxPackage *windowscommunicationsapps* | Remove-AppxPackage
Get-AppxPackage *windowscamera* | Remove-AppxPackage
Get-AppxPackage *officehub* | Remove-AppxPackage
Get-AppxPackage *skypeapp* | Remove-AppxPackage
Get-AppxPackage *getstarted* | Remove-AppxPackage
Get-AppxPackage *zunemusic* | Remove-AppxPackage
Get-AppxPackage *windowsmaps* | Remove-AppxPackage
Get-AppxPackage *solitairecollection* | Remove-AppxPackage
Get-AppxPackage *bingfinance* | Remove-AppxPackage
Get-AppxPackage *zunevideo* | Remove-AppxPackage
Get-AppxPackage *bingnews* | Remove-AppxPackage
Get-AppxPackage *onenote* | Remove-AppxPackage
Get-AppxPackage *people* | Remove-AppxPackage
Get-AppxPackage *windowsphone* | Remove-AppxPackage
Get-AppxPackage *photos* | Remove-AppxPackage
Get-AppxPackage *windowsstore* | Remove-AppxPackage
Get-AppxPackage *bingsports* | Remove-AppxPackage
Get-AppxPackage *soundrecorder* | Remove-AppxPackage
Get-AppxPackage *bingweather* | Remove-AppxPackage
Get-AppxPackage *xboxapp* | Remove-AppxPackage
```


Reinstalll default apps

`Get-AppxPackage -AllUsers| Foreach {Add-AppxPackage -DisableDevelopmentMode -Register “$($_.InstallLocation)\AppXManifest.xml”}`

**Comment by mzomparelli** [comment]
I've searched everywhere and I can't find info as to why the Win32 `EnumWindows()` returns multiple handles for a single Microsoft Edge window. If any cares to help me figure that out I would greatly appreciate it.

**Comment by mzomparelli** [comment]
Should be fixed in 1.0.43
